core/src/main/resources/benchmark-continuous-query.xml

```diff
@@ -0,0 +1,90 @@
+<benchmark xmlns="urn:radargun:benchmark:2.2">
```

Comment:
Could you please fix namespace version (3.0)? This will require some renaming as https://github.com/radargun/radargun/pull/236 has been integrated.